easybuild/main.py

```diff
@@ -600,9 +603,11 @@ def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
     if '_' not in os.environ:
         os.environ['_'] = sys.executable
 
+    if any([args, logfile, testing]):
```

Comment:
I would also do this if either `eb_go` or `cfg_settings` are `None`, so if someone calls `main()` it still works as before?